<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>46</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="state_autorize_block">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{
	border:none;
	background: #FFFFFF;
	border-radius: 16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="registrate_radiobutton">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{}

QRadioButton:checked{color:#292A2F;}

QRadioButton:unchecked{color: rgba(41, 42, 47,76 );}

QRadioButton::indicator{
	border:none;
	background:transparent;
}

</string>
              </property>
              <property name="text">
               <string>Create a free account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="styleSheet">
               <string notr="true">QFrame{
margin-top:15px;
margin-bottom:15px;
}</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="login_radiobutton">
              <property name="styleSheet">
               <string notr="true">QRadioButton{
text-align: center;
}

QRadioButton:checked{color:#292A2F;}

QRadioButton:unchecked{color: rgba(41, 42, 47,76 );}

QRadioButton::indicator{
	border:none;
	background:transparent;
}

</string>
              </property>
              <property name="text">
               <string>Sign in</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="background_login">
         <property name="styleSheet">
          <string notr="true">QGroupBox{
	border:none;
	background: #FFFFFF;
	border-radius: 16px;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>23</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	text-align: center;
}</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to a better BOT-systems&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="server_addr_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(64,64,64);
	border: 1px solid rgba(204, 204, 204, 89);
	border-radius: 5px;
	margin-left: 10px;
	margin-right: 10px;
}</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Enter your server address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(64,64,64);
	border: 1px solid rgba(204, 204, 204, 89);
	border-radius: 5px;
	margin-left: 10px;
	margin-right: 10px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Enter your email address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777208</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(64,64,64);
	border: 1px solid rgba(204, 204, 204, 89);
	border-radius: 5px;
	margin-left: 10px;
	margin-right: 10px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Enter your username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(64,64,64);
	border: 1px solid rgba(204, 204, 204, 89);
	border-radius: 5px;
	margin-left: 10px;
	margin-right: 10px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Enter your password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="confirm_password_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(64,64,64);
	border: 1px solid rgba(204, 204, 204, 89);
	border-radius: 5px;
	margin-left: 10px;
	margin-right: 10px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Confirm your password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="save_my_password">
            <property name="text">
             <string>Save my password</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="sign_up_user_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color:white;
	border: none;
	border-radius:16px;
	background-color: #4DAAFF;
}

QPushButton:!enabled{
	background-color: rgba(82,136,193,76);
}

QPushButton::hover{
	background-color: #5288C1;
}</string>
            </property>
            <property name="text">
             <string>Sign up -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="login_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color:white;
	border: none;
	border-radius:16px;
	background-color: #4DAAFF;
}

QPushButton:!enabled{
	background-color: rgba(82,136,193,76);
}

QPushButton::hover{
	background-color: #5288C1;
}</string>
            </property>
            <property name="text">
             <string>Log in -&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>45</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
